backend/app/views.py

```diff
@@ -83,3 +83,35 @@ def example_id(request, example_id):
     }
 
     return render(request, 'index.html', context)
+
+
+@api_view(['GET'])
+def all_documents(request):
+    doc_objs = Document.objects.all()
+    serializer = DocumentSerializer(doc_objs, many=True)
+    return Response(serializer.data)
+
+def documents(request):
+    """
+    All Documents page
+    """
+
+    context = {
+        'page_metadata': {
+            'title': 'Documents'
+        },
+        'component_name': 'Documents'
+    }
+
+    return render(request, 'index.html', context)
+
+@api_view(['POST'])
+def add_text(request):
+    """
+    API endpoint for adding a piece of text
+    """
+    new_text_obj = Document()
+    new_text_obj.save()
+    new_text_obj.update_metadata(request.data)
```

Comment:
Since it's PR #11's job to figure out how best to handle what happens on `Document` initialization (do we tokenize then?, etc.), we can simply skip line 115, pass `request.data` properties into the `Document` initialization in line 113, and merge this PR in. We'll need to come back to this function when we figure out exactly what a `Document` needs, but if we make this change now, we won't have any blockers on getting the rest of this code shipped.

Reply:
fixed!